src/integration-test/java/fi/hsl/common/transitdata/MockDataUtils.java

```diff
@@ -112,4 +124,152 @@ public static void setIsViaPoint(PubtransTableProtos.Common.Builder builder, boo
         else
             builder.setType(1);
     }
+
+    public static PubtransTableProtos.ROIArrival mockROIArrival(long dvjId, String routeName, long targetDateTimeMs) throws Exception {
+        return mockROIArrival(dvjId, routeName, MOCK_DIRECTION_ID, generateValidStopSequenceId(), targetDateTimeMs);
+    }
+
+    public static PubtransTableProtos.ROIArrival mockROIArrival(long dvjId, String routeName, int joreDirection, int stopSequence, long targetDateTimeMs) throws Exception {
+        PubtransTableProtos.Common common = generateValidCommon(dvjId, stopSequence, targetDateTimeMs).build();
+        PubtransTableProtos.DOITripInfo info = mockDOITripInfo(dvjId, routeName, joreDirection);
+        return mockROIArrival(common, info);
+    }
+
+    public static PubtransTableProtos.ROIArrival mockROIArrival(PubtransTableProtos.Common common, PubtransTableProtos.DOITripInfo info) throws Exception {
+        PubtransTableProtos.ROIArrival.Builder builder = PubtransTableProtos.ROIArrival.newBuilder();
+        builder.setSchemaVersion(builder.getSchemaVersion());
```

Comment:
What does this do?

Reply:
we add the default-value for the schema in the protobuf file. syntax:

```   
required int32 SchemaVersion = 1 [default = 1];
```

this already sets the value to be 1 for the getSchemaVersion(), but the generated java code however is stupid and doesn't actually raise the required-flag for these default fields, so when we call .build() -> throws exception, required field not set.  therefore we still need to explicitly set it to raise the required-flag